src/Components/PhoneChart.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import '../style/chart.css';
+
+export default class Header extends React.Component {
```

Comment:
Name of file is `PhoneChart` and name of class is `Header`. Is it supposed to be so?